projects/standard-rulesets/src/examples/requireValidExamples.ts

```diff
@@ -48,7 +48,13 @@ export function validateSchema(
   const result = schemaCompiled(example);
 
   if (!result) {
-    const error = `- ${ajv.errorsText(schemaCompiled.errors, {
+    schemaCompiled.errors?.forEach((error) => {
```

Comment:
I think we've got similar logic here https://github.com/opticdev/optic/blob/main/projects/openapi-io/src/validation/validator.ts#L67-L78

Do we want to move this into an `openapi-utilities` folder and report on AJV errors consistently?

Reply:
I looked into that, but it's very coupled with the sourcemap stuff. I think this is right for now.